src/main/java/com/example/onlinebookstore/dto/CreateBookRequestDto.java

```diff
@@ -0,0 +1,17 @@
+package com.example.onlinebookstore.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+
+@Data
+public class CreateBookRequestDto {
```

Comment:
Very long name, maybe CreateBookDto or BookRequestDto will be ok

Reply:
Yes, maybe to long. But this naming required in task.
"Add BookController with methods:
- public BookDto createBook(**CreateBookRequestDto** bookDto)".

Leave unchanged.